guides/common/modules/ref_predefined-roles.adoc

```diff
@@ -7,48 +7,140 @@ To view the exact set of permissions a predefined role grants, display the role
 For more information, see xref:Viewing_Permissions_of_a_Role_{context}[].
 
 .Permissions provided by role
-[cols="2,7" options="header"]
+[cols="3,5,5" options="header"]
 |====
-|Role |Permissions Provided by Role
+|Role |Permissions the role provides |Additional information
+
+|Access Insights Admin
+|Add and edit Insights rules.
+|
+
+|Access Insights Viewer
+|View Insight reports.
+|
+
+|Ansible Roles Manager
+|Play roles on hosts and host groups.
 
-| Access Insights Admin | Add and edit Insights rules.
-| Access Insights Viewer | View Insight reports.
-| Ansible Roles Manager | Play roles on hosts and host groups.
 View, destroy, and import Ansible roles.
+
 View, edit, create, destroy, and import Ansible variables.
-| Ansible Tower Inventory Reader | View facts, hosts, and host groups.
-| Bookmarks manager | Create, edit, and delete bookmarks.
-| Boot disk access | Download the boot disk.
-| Compliance manager | View, create, edit, and destroy SCAP content files, compliance policies, and tailoring files.
+|
+
+|Ansible Tower Inventory Reader
+|View facts, hosts, and host groups.
+|
+
+|Bookmarks manager
+|Create, edit, and delete bookmarks.
+|
+
+|Boot disk access
+|Download the boot disk.
+|
+
+|Compliance manager
+|View, create, edit, and destroy SCAP content files, compliance policies, and tailoring files.
+
 View compliance reports.
-| Compliance viewer | View compliance reports.
-| Create ARF report | Create compliance reports.
-| Default role | The set of permissions that every user is granted, irrespective of any other roles.
-| Discovery Manager| View, provision, edit, and destroy discovered hosts and manage discovery rules.
-| Discovery Reader | View hosts and discovery rules.
-| Edit hosts | View, create, edit, destroy, and build hosts.
-| Edit partition tables | View, create, edit and destroy partition tables.
-| Manager | View and edit global settings.
-| Organization admin | An administrator role defined per organization.
+|
+
+|Compliance viewer
+|View compliance reports.
+|
+
+|Create ARF report
+|Create compliance reports.
+|
+
+|Default role
+|View tasks and jobs invocations.
```

Comment:
While this is true in Satellite scenario (and maybe Orcharhino as well, I'm not sure what plugins are enabled there by default), in pure Foreman deployments one may not have remote execution or tasks plugins enabled and then they will not have those permissions. In vanilla Foreman without any plugins, the role shouldn't give any permissions, it is mostly a placeholder.

Reply:
@maximiliankolb Can you please take a look and let me know what permissions this role provides in Orcharhino?

@adamruzicka It seems I should hide this permission for Foreman builds, is that right? Or am I misunderstanding something here?

Reply:
orcharhino _always_ contains the remote execution plug-in. The default role has "view_job_invocations" and "view_foreman_tasks" permissions.

Reply:
Katello depends on REX, so you can assume it's always present in `katello,orcharhino,satellite`. In `foreman-deb,foreman-el` it's optional.

Reply:
Doesn't it optionally depend on rex?

Reply:
It's in the gemspec: https://github.com/Katello/katello/blob/f9ce129c658d6405c5d3b47dcd5cb63b27417cff/katello.gemspec#L37

Reply:
Indeed, I stand corrected.

Reply:
Thanks all for the input.

- For foreman-deb and foreman-el, the description now states that the permissions provided by Default role depend on plug-ins.
- For other builds, no change -- so View tasks and jobs invocations.

Does this work, or have I got it wrong?